worker/src/main/java/gov/cms/ab2d/worker/service/FileServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package gov.cms.ab2d.worker.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static java.nio.file.StandardOpenOption.APPEND;
+
+@Slf4j
+@Component
+public class FileServiceImpl implements FileService {
+
+
+    @Override
+    public Path createDirectory(Path outputDir) {
+        Path outputDirectory = null;
+        try {
+            outputDirectory = Files.createDirectories(outputDir);
+        } catch (IOException e) {
+            final String errMsg = "Could not create output directory : ";
+            log.error("{} : {}", errMsg, outputDir.toAbsolutePath());
+            throw new UncheckedIOException(errMsg + outputDir.getFileName(), e);
+        }
+        return outputDirectory;
+    }
+
+
+    @Override
+    public Path createFile(final Path outputDir, final String filename) {
+        final Path filePath = Path.of(outputDir.toString(), filename);
+        Path outputFile = null;
+        try {
+            //Delete file from previous run - Allows for a job can be restarted on failure.
+            Files.deleteIfExists(filePath);
```

Comment:
You should check the result of this since it returns a boolean and then handle the error if it's false